postfix/datadog_checks/postfix/data/conf.yaml.example

```diff
@@ -57,3 +57,21 @@ instances:
     tags:
       - optional_tag3
       - optional_tag4
+
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file and channel if windows_event
```

Comment:
> Set path if type is file and channel if windows_event

That is super confusing. Can you elaborate?

Reply:
it's described in the log documentation, but depending of the type, the next parameter changes

groups of configuration are: 

```
 - type: file
    path: /EZF/EZF/EF
```

```
   - type: tcp
      port: 1234
```

```
   - type: udp
      port: 1234
```

```
   - type: windows_evnet
      path: /ZEA/ZEAR/ZAER
```

Reply:
```suggestion
## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel if type is windows_event
```